README.md

```diff
@@ -0,0 +1,29 @@
+# Methods project

```

Comment:
I think you can rename it into a more relational name. Something like. `Enumerable Methods Project`